app/src/main/java/com/infomaniak/drive/data/cache/FileController.kt

```diff
@@ -656,7 +656,7 @@ object FileController {
                 result = realm.downloadAndSaveFiles(
                     localFolderProxy = folderProxy,
                     order = order,
-                    loadNextPage = loadNextPage,
+                    isFirstPage = !loadNextPage,
```

Comment:
Won't this cause problems ? because you didn't inverted the check in if after that